Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesAction.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.commonfilesearch;
+
+import java.awt.event.ActionEvent;
+import java.beans.PropertyChangeEvent;
+import java.util.EnumSet;
+import org.openide.util.HelpCtx;
+import org.openide.util.NbBundle;
+import org.openide.util.actions.CallableSystemAction;
+import org.sleuthkit.autopsy.casemodule.Case;
+import org.sleuthkit.autopsy.directorytree.FileSearchProvider;
+
+/**
+ *
+ * @author bsweeney
+ */
```

Comment:
We do not do these types of class headers, please remove authorship and provide description.